src/index.php

```diff
@@ -0,0 +1,305 @@
+     * @codeCoverageIgnore
+     */
+    function viewHandler()
+    {
+        add_thickbox();
+        ?>
+        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.min.js"
+                type="text/javascript"></script>
+        <script>
+            window.sb_nonce = "<?php echo wp_create_nonce(self::nonce); ?>";
+            -(function () {
+                var footerLinksApp = angular.module('footerLinksApp', []);
+                footerLinksApp.controller('NotificationsCtrl', function ($scope, $rootScope, $timeout) {
+                    $scope.notifications = [];
+                    $rootScope.$on('notification', function (event, data) {
+                        $scope.notifications.push(data);
+                        $timeout(function () {
+                            $scope.removeNotification(data);
+                        }, 10000);
+                    });
+                    $scope.removeNotification = function (notification) {
+                        var index;
+                        if ($scope.notifications !== undefined) {
+                            index = $scope.notifications.indexOf(notification);
+                            $scope.notifications.splice(index, 1);
+                        }
+                    }
+                });
+
+                footerLinksApp.controller('MajikCtrl', function ($scope, $http, $rootScope) {
+                    var footerClass = null;
+                    var wp_ajax = function (_req) {
+                        _req.nonce = window.sb_nonce;
+                        return $http({
+                            method: 'POST',
+                            url: ajaxurl,
+                            data: jQuery.param(_req),
+                            headers: {'Content-Type': 'application/x-www-form-urlencoded'}
+                        })
+                    };
+                    $scope.patchTheme = function () {
+                        wp_ajax({
+                            action: 'NFL-patch_theme',
+                            data: $scope.footerClass
+                        }).success(function (res) {
+                            if (res.success) {
+                                $rootScope.$emit('notification', {
+                                    state: 'success',
+                                    msg: res.data
+                                });
+                            } else {
+                                $rootScope.$emit('notification', {
+                                    state: 'errror',
+                                    msg: res.data
+                                });
+                            }
+                        });
+                    };
+                });
+            })(angular, document, jQuery);
+        </script>
+        <h1>Remove Footer Links</h1>
+        <hr/>
+        <div ng-app="footerLinksApp">
+            <div ng-controller="MajikCtrl">
+                <h2>Input CSS class</h2>
```

Comment:
I was wondering if we could add this into the theme preview and use their apis? New wp 4.7 also supports custom css for all themes directly from theme selector, can we use that?